Mars Mission/Mars Mission/Model/WeatherModel.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  WeatherModel.swift
+//  Mars Mission
+//
+//  Created by Aldrin Bahle Gama on 2020/11/04.
+//
+
+import Foundation
+
+struct WeatherDataModel: Codable {
```

Comment:
Make the variables in the structs optional and use them safely

Reply:
<img width="1131" alt="Screenshot 2020-11-05 at 05 41 12" src="https://user-images.githubusercontent.com/20769103/98195073-90e55180-1f29-11eb-82d1-a4fe2a7c6c8b.png">

fixed.